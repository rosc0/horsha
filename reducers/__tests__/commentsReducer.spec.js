import reducer, { initialState as state } from '../comments'
// import types from '../../constants/actionTypes'

// import { horse_picture, horse_comments } from '../../../test/api'

describe('Comments Reducer', () => {
  it('should return initial state', () => {
    expect(reducer()).toBe(state)
    expect(reducer()).toMatchSnapshot()
  })
})
