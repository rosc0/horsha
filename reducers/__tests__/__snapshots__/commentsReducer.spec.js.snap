// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments Reducer should return initial state 1`] = `
Object {
  "collection": Array [],
  "error": false,
  "fetched": false,
  "fetching": false,
  "next": null,
  "previous": null,
  "remaining": 0,
  "toggle": false,
}
`;
